src/aleph/vm/orchestrator/views/__init__.py

```diff
@@ -337,11 +337,11 @@ async def status_public_config(request: web.Request):
     )
 
 
-def authenticate_api_request(request: web.Request) -> bool:
+def authenticate_api_request(request: web.Request, raises_on_missing_header=True) -> bool:
```

Comment:
What about making this the default behavior and skipping the extra argument ?

Reply:
The update allocation endpoint rely on that behaviour